docs/resources/cluster.md

```diff
@@ -113,7 +113,7 @@ Required:
 
 Optional:
 
-- `node_count` (Number) Number of nodes in the region. Will always be 0 for serverless clusters.
+- `node_count` (Number) Number of nodes in the region. Valid for Advance clusters only.
```

Comment:
@kathancox this one should be pretty straightforward.  Just this line to check. Thank you!